airbyte-cdk/python/airbyte_cdk/destinations/vector_db_based/embedder.py

```diff
@@ -91,7 +104,7 @@ def check(self) -> Optional[str]:
             return format_exception(e)
         return None
 
-    def embed_chunks(self, chunks: List[Chunk]) -> List[List[float]]:
+    def embed_chunks(self, chunks: List[Chunk], max_token_length: int) -> List[List[float]]:
         return self.embeddings.embed_documents([chunk.page_content for chunk in chunks])
```

Comment:
This looks like a bug because `max_token_length` is not used. If that's the desired behavior, can you note in a docstring that we either don't need to handle the max_token_length, or else we are handling it elsewhere?